data/core/units/dunefolk/Wyvern_Rider.cfg

```
@@ -14,7 +14,7 @@
     {AMLA_DEFAULT}
     cost=102
     usage=scout
-    description=""
+    description= _ "Wyverns are fierce and angry beasts, especially when provoked. Most travellers would run or fight in their mighty presence. Yet a few are able to tame these wild creatures. These dunefolk riders spend years, even decades of their lives bonding with creatures. They skillfully dedicate their lives to art of flying the Wyvern. While these masters do not usually use this art in warfare, they have been known to become threatening warriors when defending their home."
```

Preliminary suggested changes:

* Merge sentences 2 and 3 into a single sentence, with a comma
* Add an article for "creatures"
* Make "home" plural
* Don't capitalize the race name

The result:
```suggestion
    description= _ "Wyverns are fierce and angry beasts, especially when provoked. Most travellers would run or fight in their mighty presence, yet a few are able to tame these wild creatures. These dunefolk riders spend years, even decades of their lives bonding with the creatures. They skillfully dedicate their lives to art of flying the wyvern. While these masters do not usually use this art in warfare, they have been known to become threatening warriors when defending their homes."
```

I think it still has some problems even with these changes. In particular, the sentence with "skillfully" feels awfully abrupt.